provider/rfc2136/rfc2136.go

```diff
@@ -209,30 +216,32 @@ func (r rfc2136Provider) List() ([]dns.RR, error) {
 		return make([]dns.RR, 0), nil
 	}
 
-	log.Debugf("Fetching records for '%s'", r.zoneName)
+	records := make([]dns.RR, 0)
+	for _, zone := range r.zoneNames {
+		log.Debugf("Fetching records for '%s'", zone)
```

Comment:
'%s' -> %q , if you move the code make it better :)

Reply:
fixed: https://github.com/kubernetes-sigs/external-dns/pull/3976/commits/0ac4740162c64059a8557f5226b2609f43c20f99